modules/KIWILinuxRC.sh

```diff
@@ -1324,8 +1324,10 @@ EOF
     # install grub2 in BIOS mode
     #--------------------------------------
     if [ ! -z "$kiwi_OfwGrub" ];then
+        local prepdev=$imagePrepDevice
+        test -z $prepdev && prepdev=$(ddn $imageDiskDevice $kiwi_OfwGrub)
```

Comment:
I believe you that your code works, but why do we want to mess with a variable which is not needed at all. Your code adds

```
test -z $prepdev && prepdev=$(ddn $imageDiskDevice $kiwi_OfwGrub)
```

but test -z $prepdev is always true because suse-preinit never exports imagePrepDevice
Thus why not simply delete the imagePrepDevice mess

This is not about your code it's about my crappy code :)